### ISSUE TITLE
no security for geofence webhook

### ISSUE THREAD

**Issue by richie510** [PeteLawrence/homebridge-people#49]
I'm a little concerned about opening up a port for this without any security. Is there something I'm missing to make this more secure, or am I just worrying for nothing?

**Comment by benzman81** [comment]
you are right, no otherway to push something to your local homebridge.

**Comment by benzman81** [comment]
You can just poll for the information to another server, but this must be OPEN as well. Locative has an API for example, but stressing their server with polling might not be a good idea.

**Comment by JonSilver** [comment]
Of course, it could also subscribe to selected mqtt topics where presence information is published, which is less of a hole… but requires a whole bunch of additional infrastructure as well as an mqtt-enabled geolocator.

**Comment by benzman81** [comment]
I dont know mqtt much, but first read I saw that you need to open a port on your router as well.

**Comment by JonSilver** [comment]
It depends... if it's running inside your firewall then yes of course you have to open a port. There are MQTT broker services you can hire too, if you'd prefer to leave the firewall stuff to someone else! :)

My MQTT servers run inside our firewalls on Pi's with TLS end to end encryption. I'm happy with that level of security.
________________________________
